db/migrate/20160107120927_add_cron_table.rb

```diff
@@ -0,0 +1,16 @@
+class AddCronTable < ActiveRecord::Migration
+
+  def up
+    execute "CREATE TYPE cron_type AS ENUM ('daily', 'weekly', 'monthly');"
+    create_table :crons do |t|
+      t.references :branch
+      t.column :type, :cron_type
```

Comment:
sorry, I might have not explained myself, this should just be a string as there is no postgres type of `cron_type`